_posts/en/newsletters/2024-05-17-newsletter.md

```diff
@@ -0,0 +1,170 @@
+  proof-of-concept [forum][hodlboard] that requires providing a autct
+  proof in order to sign up, providing an environment where everyone is
+  known to be a holder of bitcoins but no one needs to provide any
+  identifying information about themselves or their bitcoins.
+
+- **BIP39 seed phrase splitting:** Rama Gan [posted][gan penlock] to the
+  Bitcoin-Dev mailing list a link to a [set of tools][penlock website]
+  they have developed for generating and splitting a [BIP39][] seed
+  phrase without using any electronic computing equipment (except to
+  print instructions and templates).  This is similar to [codex32][topic
+  codex32] but operates on BIP39 seed words that are compatible with
+  almost all current hardware signing devices and many software wallets.
+
+  Andrew Poelstra, co-author of codex32, [replied][poelstra penlock1]
+  with several comments and suggestions.  Without us trying both
+  schemes---which would take several hours each---the exact set of
+  tradeoffs between them isn't clear to us.  However, both seem to offer
+  the same fundamental capabilities: instructions for securely
+  generating a seed offline; the ability to split the seed into multiple
+  shares using [Shamir's secret sharing][sss]; the ability to
+  reconstitute the shares into the original seed; and the ability to
+  verify checksums on both the shares and the original seed, allowing
+  users to detect data corruption early when the original data might
+  still be recoverable.
+
+- **Alternative to BitVM:** Sergio Demian Lerner and several co-authors
+  [posted][lerner bitvmx] to the Bitcoin-Dev mailing list about a new
+  virtual CPU architecture based in part on the ideas behind
+  [BitVM][topic acc].  The goal of their project, BitVMX, is to be able
+  to efficiently prove the proper execution of any program that can be
+  compiled to run on an established CPU architecture, such as
+  [RISC-V][].  Like BitVM, BitVMX does not require any consensus changes
+  but it does require one or more designated parties act as a trusted
+  verifier.  That means multiple users interactively participating in a
+  contract protocol can prevent any one (or more) of the parties from
+  withdrawing money from the contract unless that party successfully
+  executes an arbitrary program specified by the contract.
+
+  Lerner links to a [paper][bitvmx paper] about BitVMX which compares it
+  to the original BitVM (see [Newsletter #273][news273 bitvm]) and to
+  the limited details available about follow-up projects from the
+  original BitVM developers.  An accompanying [website][bitvmx website]
+  provides additional information in a slightly less technical form.
+
+- **Continued discussion about updating BIP2:** Mark "Murch" Erhardt
+  [continued][erhardt bip2] the discussion on the Bitcoin-Dev mailing
+  list about updating [BIP2][], which is the document that currently
+  describes the Bitcoin improvement proposals (BIP) process.  His email
```

Comment:
```suggestion
  describes the Bitcoin Improvement Proposals (BIP) process.  His email
```

Reply:
Leaving as-is.  Up until about a year ago, we did uppercase words before introducing an abbreviation for them.  However, when I did that for Mastering Bitcoin, my editor ended up changing them all to lowercase.  I thought about it for a bit and decided that there really is no good reason to uppercase the expanded forms, so I switched to that style here also.